common/src/chain/config.rs

```diff
@@ -82,17 +84,19 @@ fn create_mainnet_genesis() -> Block {
     use crate::chain::transaction::{Destination, TxInput, TxOutput};
     use crate::primitives::Amount;
 
+    // TODO: replace this with our mint key
+    let (_mint_priv_key, mint_pub_key) = PrivateKey::new(KeyKind::RistrettoSchnorr);
+    let genesis_mint_receiver_pubkeyhash = PublicKeyHash::from(&mint_pub_key);
+
     let genesis_message = b"".to_vec();
-    let genesis_mint_receiver = Address::new_with_hrp(MAINNET_ADDRESS_PREFIX, [])
-        .expect("Failed to create genesis mint address");
     let input = TxInput::new(
         Id::<Transaction>::new(&H256::zero()).into(),
         0,
-        genesis_message,
+        InputWitness::NoSignature(Some(genesis_message)),
     );
     let output = TxOutput::new(
         Amount::from_atoms(100000000000000),
```

Comment:
We should probably add a comment about this being temporary and not the final amount